src/index.html

```diff
@@ -11,7 +11,129 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form 
+      action="http://localhost:8080/" 
+      method="post" 
+      class="form">
+
+      <fieldset  class="fieldset">
+        <legend>Personal information</legend>
+        <label class="naming">
+          Surname:
+          <input type="text" name="surname">
+        </label> 
+        <label class="naming">
+          Name:
+          <input type="text" name="name">
+        </label>
+        <label class="naming">
+          How old are You?
+          <input type="number" name="age"
+          value="12" class="ageSize"
+          >
```

Comment:
code style is everything
```suggestion
          <input 
            type="number" 
            name="age"
            value="12" 
            class="ageSize"
          >
```